src/patchVNode/updateElement.ts

```diff
@@ -6,12 +6,6 @@ export function updateElement(formerVNode: VNode, vNode: VNode): ElementVNode {
   if (isElement(node)) {
```

Comment:
Again, consider using the same technique as above to eliminate nested if-statements.

Reply:
This type guard is required by TypeScript to be here like this because of the `node is Element` check must be performed in the `if`.

![image](https://user-images.githubusercontent.com/6282473/29818774-db2af3be-8c8b-11e7-9c35-61db2fa9b129.png)

Reply:
Okay.